src/materials/nodes/SpriteNodeMaterial.js

```diff
@@ -26,6 +26,7 @@ class SpriteNodeMaterial extends NodeMaterial {
 		this.isSpriteNodeMaterial = true;
 
 		this.lights = false;
+		this.useSizeAttenuation = true;
```

Comment:
I think we should make this private -- or use an underscore.

@Mugen87 What coding pattern do you prefer?

Reply:
Properties that are marked as private should use an underscore. They will migrated to private fields in the future.